workerloop.go

```diff
@@ -189,9 +189,16 @@ func (lh *WorkerLoop) ValidateBlockConsensus(ctx context.Context, block interfac
 		senderIds = append(senderIds, memberId)
 	}
 
-	isQuorum, sendersTotalWeight, q := quorum.IsQuorum(senderIds, committeeMembers)
-	if !isQuorum {
-		return errors.Errorf("ValidateBlockConsensus: sendersTotalWeight=%d is less than quorum=%d (committeeMembersCount=%d)", sendersTotalWeight, q, len(committeeMembers))
+	if softVerify {
```

Comment:
Are there scenarios where softVerify is not enough?

Reply:
The idea is to mitigate issues if the f honesty assumption fails: 
In consensus there is a liveness issue, but state being verified by all honest nodes (plus vc owner protection -controlling  state changes), would be hard to manipulate.
 In node sync vs consensus we currently do not verify state only signatures so the effect could be severe.